examples/demo-runtime-layer-function/template.yml

```diff
@@ -16,6 +16,7 @@ Resources:
   DemoPowerShellFunction:
     Type: AWS::Serverless::Function
     Properties:
+      Architectures: arm64
```

Comment:
Great to have this!
The architectures needs a list, rather than string so the syntax needs to be:
`      Architectures:
        - arm64`

Reply:
<sigh>. Gotta love that CloudFormation still deployed this with success. 😂